docs/spec/tendermint-fork-cases/MC_n10_f3.tla

```diff
@@ -0,0 +1,26 @@
+----------------------------- MODULE MC_n10_f3 -------------------------------
```

Comment:
Are these MC files generated automatically, or does one have to do copy-paste?

Reply:
These ones a produced by copy-paste :(